.github/scripts/update-instrumentation-version.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+version=$1
+
+sed -Ei "s/val otelInstrumentationVersion = \"[^\"]*\"/val otelInstrumentationVersion = \"$version\"/" dependencyManagement/build.gradle.kts
```

Comment:
Didn't check, but is this file executable?

Reply:
Checked and it is. 👍

Reply:
thanks for checking, I usually forget until it fails during the release process, but somehow remembered this time 😂